transformer_engine/pytorch/permutation.py

```diff
@@ -0,0 +1,239 @@
+# Copyright (c) 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+"""Linear API"""
+import torch
+import warnings
+from typing import Tuple
+
+import transformer_engine_torch as tex
+
+
+__all__ = [
+    "Permute",
```

Comment:
Should be lower-case?

Reply:
After discussion with @yaox12 , I changed the names to `moe_permute` and `moe_unpermute` to distinguish them from pytorch ops.